mlflow/java/client/src/main/java/org/mlflow/tracking/RunsPage.java

```diff
@@ -0,0 +1,91 @@
+package org.mlflow.tracking;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import org.mlflow.api.proto.Service.*;
+
+public class RunsPage implements Page<Run> {
+
+    private final String token;
+    private final List<Run> runs;
+
+    private final MlflowClient client;
+    private final List<String> experimentIds;
+    private final String searchFilter;
+    private final ViewType runViewType;
+    private final List<String> orderBy;
+    private final int maxResults;
+
+    /**
+     * Creates a fixed size page of Runs.
+     */
+    public RunsPage(List<Run> runs,
```

Comment:
Can we make this interface have no visibility modifier? Only visible to `package org.mlflow.tracking;`

Reply:
Done